cvat/apps/documentation/user_guide.md

```diff
@@ -645,63 +645,57 @@ Applying ``split`` into an interpolated track divides it into two separate track
+-   Show every image in full size or zoomed out like previous
+    (it is enabled by default for interpolation mode and disabled for annotation mode).
+- ``Rotate all images``  checkbox — switch the rotation of all frames /an individual frame.
+-   Adjust ``Brightness``/``Contrast``/``Saturation`` of too exposed or too
+    dark images using ``F3`` — color settings (changes displaying settings and not the
     image itself).
 
 Shortcuts:
--   ``Shift+B``/``Alt+B`` for brightness
--   ``Shift+C``/``Alt+C`` for contrast
--   ``Shift+S``/``Alt+S`` for saturation
--   ``F2`` —> ``Reset Color Settings`` to default values
+-   ``Shift+B+=``/``Shift+B+-`` for brightness.
+-   ``Shift+C+=``/``Shift+C+-`` for contrast.
+-   ``Shift+S+=``/``Shift+S+-`` for saturation.
 
-    ![](static/documentation/images/image069.jpg)
+    ![](static/documentation/images/image164_mapillary_vistas.jpg)
 
-``Other Settings`` contains:
-- ``Show All Interpolation Tracks`` checkbox — shows hidden object on the
-  side panel for every interpolated object (turned off by default)
-- ``AAM Zoom Margin`` slider — defines margins for shape in attribute annotation mode
-- ``Enable AutoSaving`` checkbox — turned off by default
-- ``AutoSaving Interval (min)`` input box — 15 minutes by default
-- ``Rotate all images``  checkbox — switch the rotation of all frames / individual frame
+-   ``Reset Color Settings`` to default values.
 
 ---
-### Bottom Panel
 
-![](static/documentation/images/image035.jpg)
+In tab ``Workspace`` you can:
 
----
-Go to the first and latest frames.
+![](static/documentation/images/image155.jpg)
 
-![](static/documentation/images/image036.jpg)
+- ``Enable Autosaving`` checkbox — turned off by default.
+- ``Autosaving Interval (min)`` input box — 15 minutes by default.
+- ``Show All Interpolation Tracks`` checkbox — shows hidden objects on the
+  side panel for every interpolated object (turned off by default).
+- ``Always show object details`` - show text for an object on the canvas not only when the object is activated:
 
----
-Go to the next/previous frame with a predefined step. Shortcuts:
-``V`` — step backward, ``C`` — step forward. By default the step is ``10``
-(change at ``Open Menu`` —> ``Settings`` —> ``Player Step``).
+   ![](static/documentation/images/image152_DETRAC.jpg)
 
-![](static/documentation/images/image037.jpg)
+- ``Automatic bordering`` - enable automatic bordering for polygons and polylines during drawing/editing. 
+  For more information To find out more, go to the section [annotation with polygons](#Annotation-with-polygons).
+- ``Attribute annotation mode (AAM) Zoom Margin`` input box — defines margins (in px) 
+  for shape in the attribute annotation mode.
+- Press `` Go back`` or ``F3`` to return to the annotation.
 
 ---
-Go to the next/previous frame with step equals to 1. Shortcuts: ``D`` — previous, ``F`` — next.
 
-![](static/documentation/images/image040.jpg)
 
 ---
```

Comment:
Should it be removed (we already have one above)?

Reply:
@nmanovic these lines should be deleted at #1509. I will fix this by adding them back to this PR and deleting in #1509